emails/2022-07-22.md

```diff
@@ -0,0 +1,46 @@
+This is Tim Sehn (tim@dolthub.com), Founder and CEO of DoltHub. We decided this week to ditch the automated or semi-automated emails you all were getting as users. We're a tiny company. We want to build a personal connection with as many of you as possible. Automated emails weren't helping with that. I'm going to write this weekly email to you all every Friday until I can't, which I hope is many years from now.
+
+# That belongs in a museum...
+
+We're making great progess on the [museum collections bounty](https://www.dolthub.com/repositories/dolthub/museum-collections). We're up to 10.6M objects in `main`.
+
+```bash
+museum-collections $ dolt sql -q "select count(*) from objects"
++----------+
+| count(*) |
++----------+
+| 10659271 |
++----------+
+```
+
+We have approximately [20 outstanding Pull Requests](https://www.dolthub.com/repositories/dolthub/museum-collections/pulls) to review and merge. The top bounty hunter is [in line to make ~$3,400](https://www.dolthub.com/repositories/dolthub/museum-collections/bounties/880c0d21-c334-4417-bc18-8be7bddfb062/scoreboard). We continue to be inspired by the bounty process' ability to generate awesome open data to drive [interesting, socially relevant analysis](https://www.dolthub.com/blog/2022-07-01-hospitals-compliance/).
```

Comment:
```suggestion
We have approximately [20 outstanding Pull Requests](https://www.dolthub.com/repositories/dolthub/museum-collections/pulls) to review and merge. The top bounty hunter is [in line to make ~$3,400](https://www.dolthub.com/repositories/dolthub/museum-collections/bounties/880c0d21-c334-4417-bc18-8be7bddfb062/scoreboard). We continue to be inspired by the bounty process's ability to generate awesome open data to drive [interesting, socially relevant analysis](https://www.dolthub.com/blog/2022-07-01-hospitals-compliance/).
```

Reply:
https://forum.wordreference.com/threads/process-or-processs.1704502/#:~:text=Thanks%20in%20advance!,-bj&text=The%20traditional%20way%20to%20form,method%20I%20prefer%20to%20use.